doc/api/cli.md

```diff
@@ -411,14 +411,18 @@ Node options that are allowed are:
 - `--enable-fips`
 - `--force-fips`
 - `--icu-data-dir`
+- `--inspect-brk`
```

Comment:
You must have had a reason to leave those out on the first run?
They will make a very weird effect...
`inspect-port` in benign if used alone.